compress_multiphoton/compression.py

```diff
@@ -48,7 +48,7 @@ def compute_quantal_size(movie: np.array) -> dict:
         max(counts_slice.stop * 20 // 100, counts_slice.start), counts_slice.stop
     )
     assert (
-        counts_slice.stop - counts_slice.start - movie.max() * 0.10 > 0
+        counts_slice.stop - counts_slice.start > 0.10 * movie.max() 
     ), f"At least intensity level {movie.max() * 0.10} is expected, got {counts_slice.stop - counts_slice.start}."
```

Comment:
@tdincer. Think of a message in the context of the user who is receiving this message without reading the code. It needs context. 
```suggestion
    ), f"The image does not have a sufficient range of intensities to compute the noise transfer function."
```